package.json

```diff
@@ -143,6 +154,10 @@
           "command": "peripheral-inspector.svd.copyValue",
           "when": "false"
         },
+        {
+          "command": "peripheral-inspector.svd.exportNode",
```

Comment:
should exportAll also be hidden from the palette?

Reply:
yes, that's right, I have hidden it from the palette.